charts/hlf-k8s/templates/job-hook-delete-secrets.yaml

```diff
@@ -26,15 +26,14 @@ metadata:
   annotations:
     "helm.sh/hook": post-delete
     "helm.sh/hook-delete-policy": hook-succeeded
-  namespace: kube-system # Since we delete the secrets after 'helm delete',
-                         # we need to use the Tiller SA in the kube-system
-                         # namespace
+  {{- if eq (default .Values.hooks.serviceAccount "") "tiller" }} # The tiller SA is located in the kube-system namespace
```

Comment:
defaulting to "" is unnecessary here as it's already defined as "tiller" in the default values. Also, I recommend using explicit over implicit here and adding a hooks.serviceAccount.namespace value that default to kube-system so you don't have to name your SA tiller if you want to put it on kube-system.